account/manager.go

```diff
@@ -548,10 +601,24 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 					"be greater than 0", acctKey)
 			}
 
-			// If we have a label prefix, then we'll apply that now
-			// and also attach some additional meta data.
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
+			// Attach additional meta data to transaction label.
+			//
+			// TODO(ffranr): Tx inputs are required to calculate the fee. The fee
+			// will be added to the tx label. m.cfg.Wallet.SendOutputs returns
+			// and selects tx inputs but also requires the label as an argument.
+			// Instead, the tx should be constructed via m.cfg.Wallet.FundPsbt
+			// which would give us an opportunity to inspect tx inputs before
+			// creating a tx label.
+			txFee := btcutil.Amount(0)
+
+			balanceDiff := account.Value
+			contextLabel, err := actionTxLabel(
+				account, CREATE, false,
```

Comment:
nit: should use the full 80 characters limit for the argument list, so I think it would fit even compacter:
```
			contextLabel, err := actionTxLabel(
				account, CREATE, false, txFee, balanceDiff,
			)
```
Same below.